tutor4points/core/templates/home.html

```diff
@@ -70,13 +100,88 @@ <h2>Suggested Sessions</h2>
+                        {% with tutor=sent_request.tutor %}
+                        <div class="info-container d-flex" id="tutor-{{tutor.id}}" data-tutorid="{{tutor.id}}" >
+                            <div>
+                                <img src="{{sent_request.tutor.profile_pic.url}}" class="profile-pic">
+                            </div>
+                            <div class="name-column">
+                                {{ tutor }} <br>
+                                <div class="classes-column">
+                                    {{ sent_request.class_name }} @ {{ sent_request.location }} <br/>
+                                </div>
+                            </div>
+                            <div class="times-column">
+                                {{sent_request.tutor_date}} @ {{sent_request.tutor_time}} <br/>
+                            </div>
+                            {% if sent_request.tutee_comment %}
+                                Comment: {{sent_request.tutee_comment}} <br> <br>
+                            {% else %}
+                                No comment <br> <br>
+                            {% endif %}
+                        </div>
+                        {% endwith %}
+                    {% endfor %}
+                </div>
+            </div>
+
+        </div>
+    </div>
+</div>
+
+{% if form.errors %}
+<script>
+    $(document).ready (function(){
+        $("#tutor-request-modal").modal('show')
+    })
+</script>
+{% endif %}
+
+
+<div class = "modal fade" id = "tutor-request-modal">
+    <div class = "modal-dialog model-lg">
+        <div class= "modal-content">
+           <div class = "modal-flex-container">
+               <div class = "form-container">
+                    <form method="POST" class="form-group" enctype='multipart/form-data'>
+                        {% csrf_token %}
+                        <input type="hidden" name="requestedTutor" id="requestedTutor" value="">
+                        
+                        <div class="form-row">
+                            {{form.class_name | as_crispy_field}}
+                        </div>
+                        <div class="form-row">
+                            {{form.tutor_date | as_crispy_field}}
+                        </div>
+                        <div class="form-row">
+                            {{form.tutor_time | as_crispy_field}} 
+                        </div>
+                        <div class="form-row">
+                            {{form.location | as_crispy_field}}
+                        </div>
+                        <div class="form-row">
+                            {{form.tutee_comment | as_crispy_field}}
+                        </div>
+
+                        <button type="submit" class="btn-block btn-secondary btn-style">Send Request</button>
+                    </form>
+               </div>
+           </div> 
         </div>
     </div>
 </div>
 
-<script type="text/javascript">
```

Comment:
It will be helpful to have the boxes lead to the requests page rather than the user's profile. Or maybe add an accept/decline button like the one on the "Tutor request" page rather than leading it to a different page.

Reply:
For the 'Tutor Requests' component, I can redirect to the requests page. How does that sound?

Reply:
Sounds good to me.